docs/core.yml

```diff
@@ -2183,6 +2192,190 @@ components:
+          additionalProperties:
+            type: integer
+            format: int32
+        configInheritanceCounts:
+          type: object
+          additionalProperties:
+            type: integer
+            format: int32
+        creationDate:
+          type: string
+          example: Mon Feb 22 08:40:43 UTC 2021
+        modificationDate:
+          type: string
+          example: Mon Feb 22 08:40:43 UTC 2021
+        deactivationDate:
+          type: string
+          example: Mon Feb 22 08:40:43 UTC 2021
+        settings:
+          $ref: "#/components/schemas/OrgSettingsData"
+        settingsInherited:
+          $ref: "#/components/schemas/OrgSettingsData"
+        settingsSummary:
+          $ref: "#/components/schemas/OrgSettingsData"
+        registrationKey:
+          type: string
+        reporting:
+          $ref: "#/components/schemas/OrgReportingData"
+        customConfig:
+          type: boolean
+    OrgReportingData:
+      type: object
+      properties:
+        orgManagers:
+          type: array
+          items:
+            $ref: "#/components/schemas/User"
+    OrgSettingsData:
+      type: object
+      properties:
+        maxSeats:
+          type: integer
+          format: int32
+        maxBytes:
+          type: integer
+          format: int64
+    TenantUid:
+      type: object
+      properties:
+        defaultSystemTenant:
+          type: boolean
+    User:
+      type: object
+      properties:
+        tenantUid:
+          $ref: "#/components/schemas/TenantUid"
+        principalId:
+          type: string
+        userId:
+          type: integer
+          format: int32
+        orgId:
+          type: integer
+          format: int32
+        username:
+          type: string
+        email:
+          type: string
+        firstName:
+          type: string
+        lastName:
+          type: string
+        lastFirstName:
+          type: string
+        lastFirstSearch:
+          type: string
+        firstLastSearch:
+          type: string
+        question:
+          type: string
+        answer:
+          type: string
+        passwordResetRequired:
+          type: boolean
+        active:
+          type: boolean
+        blocked:
+          type: boolean
+        maxBytes:
+          type: integer
+          format: int64
+        ssoIdentityProviderUid:
+          type: integer
+          format: int64
+        modificationDate:
+          type: string
+          format: date-time
+        creationDate:
+          type: string
+          format: date-time
+        apiTokenRevocationTime:
+          type: integer
+          format: int64
+        externalReference:
+          type: string
+        localAuthenticationOnly:
+          type: boolean
+        supportUser:
+          type: boolean
+        status:
+          type: string
+        tableName:
+          type: string
+        idColumnName:
+          type: string
+        principalType:
+          type: string
```

Comment:
it is odd to me that swagger uses type: String with enum https://swagger.io/docs/specification/data-models/enums/
but LGTm